### PR TITLE
Profile UI

### PR DESCRIPTION
**Pull request by bellalyuxe**

### CODE REVIEW

**Comment by alex-berezovsky** [comment]
Looks awesome.

**Comment by joanna985** [comment]
I like it. Great job.

File: app/src/screens/ProfileScreen.tsx [hunk 1]
```diff
@@ -94,9 +94,11 @@ export default function ProfileScreen({
         >
           <Text style={styles.label}>
             {" "}
-            Name:{" "}
+            <Text style={styles.subtitle}> 
+              Name{" "}
```
Comment (SugjuChoi) [inline app/src/screens/ProfileScreen.tsx:98]: Good change in the text style

File: app/src/screens/ProfileScreen.tsx [hunk 2]
```diff
@@ -219,10 +247,11 @@ const styles = StyleSheet.create({
     paddingVertical: 8,
     backgroundColor: "#DBECEE",
     paddingHorizontal: 15,
-    borderRadius: 4,
+    borderRadius: 10,
```
Comment (SugjuChoi) [inline app/src/screens/ProfileScreen.tsx:250]: Good UI change. Makes it look better.

**Review by SugjuChoi — COMMENTED** [review]
Great UI changes in general.

**Comment by joanna985** [comment]
Looks great!

**Review by sredford6 — COMMENTED** [review]
The changes to the UI look nice.

**Pull request merged**